docs/content/quickstart.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Quickstart Guide
+descriptions: Get started using Porter
+---
+
+## Getting Porter
+
+First make sure Porter is installed.
+Please see the [installation instructions](./install.md) for more info.
+
+## Creating a new installer
```

Comment:
This should be `Create a new bundle`.

We haven't made a decision to not use the same term as [CNAB](https://cnab.io) for bundles. So while informally on the landing page we called it a cloud installer, we aren't ready to change the terms everywhere. It would be confusing I think. @jeremyrickard ?